media/css/firefox/tiles.less

```diff
@@ -109,7 +109,17 @@
 }
 
 @media only screen and (min-width: @breakTablet) and (max-width: @breakDesktop) {
-
+ 
+    #main-feature h1{
+        .font-size(64px);
+        float: none;
+        width: auto;
+    }
+    
+    .main-column{
+        width: auto;
```

Comment:
Just adjusting the width alone doesn't do the trick here, as the element is still `float`ed to the left. This leaves content running off the right edge of the window.

<img width="826" alt="screen shot 2015-09-18 at 5 58 01 pm" src="https://cloud.githubusercontent.com/assets/317532/9972862/9d904de2-5e2f-11e5-9df0-690418b702e1.png">

You'll need to remove the float on this element - `float: none;`